ci-operator/step-registry/idp/htpasswd/idp-htpasswd-commands.sh

```diff
@@ -106,20 +108,33 @@ function set_users () {
     jq $IDP_FIELD' += [{"htpasswd":{"fileData":{"name":"cucushift-htpass-secret"}},"challenge":"true","login":"true","mappingMethod":"claim","name":"cucushift-htpasswd-provider","type":"HTPasswd"}]' "${oauth_file_src}" > "${oauth_file_dst}"
     oc replace -f "${oauth_file_dst}"
 
-    # wait for oauth-openshift to rollout
-    wait_auth=true
+    echo "Wait up to 10 minutes for htpasswd ready"
+    auth_ready=false
+    count=0
     expected_replicas=$(oc get deployment oauth-openshift -n "$OAUTH_NAMESPACE" -o jsonpath='{.spec.replicas}')
-    while $wait_auth;
+    while [[ $count -lt 40 ]]
     do
         available_replicas=$(oc get deployment oauth-openshift -n "$OAUTH_NAMESPACE" -o jsonpath='{.status.availableReplicas}')
         new_gen=$(oc get deployment oauth-openshift -n "$OAUTH_NAMESPACE" -o jsonpath='{.metadata.generation}')
         if [[ $expected_replicas == "$available_replicas" && $((new_gen)) -gt $((gen)) ]]; then
-            wait_auth=false
+            auth_ready=true
+            break
         else
-            sleep 10
-        fi
+	    echo "waiting 15s now. elapsed: $(( 15 * $count )) seconds"
```

Comment:
Your latest update https://github.com/openshift/release/compare/3a1af20be8033a2718c66cb720f6c511b7b66ab4..f4e79945ec1b2769f729a6efe1f1f47efc7ff4ce introduced new issue: it includes Tab character at the beginning. It's better to use consistent spaces.

Reply:
Fixed